serverless/messaging/reference-content/nats-overview.mdx

```diff
@@ -19,15 +19,15 @@ NATS (**N**eural **A**utonomic **T**ransport **S**ystem) is a messaging system t
 
 NATS is a [Cloud Native Computing Foundation](https://www.cncf.io/) project supported by a strong community, and can also be integrated in a Kubernetes Cluster.
 
-## NATS namespaces
+## NATS accounts
 
-You can easily create a NATS namespace from the [Scaleway console](/serverless/messaging/how-to/create-namespace), via the [Scaleway API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) or via [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/guides/mnq_with_nats_terraform_provider).
+You can easily create a NATS account from the [Scaleway console](/serverless/messaging/how-to/get-started/#how-to-create-a-nats-account), via the [Scaleway API](https://www.scaleway.com/en/developers/api/messaging-and-queuing/) or via [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/guides/mnq_with_nats_terraform_provider).
```

Comment:
I would expect to be redirected to the Scaleway console if the clickable text is "Scaleway console", maybe move the link to "create a NATS account"?

Reply:
Modified